service2/server.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const axios = require("axios");
+const app = express();
+
+const PORT = process.env.PORT || 3001;
+
+// This says to listen for a GET request on the route /aRoute'
+app.get("/aRoute", (req, res) => {
+  // This will perform a GET request to service1
+  axios
+    .get("http://localhost:3000/aRoute")
```

Comment:
Should make the host (`http://localhost:3000`) be able to be overridden as an environment variable like port

Reply:
> It'd be great if you could get this deployable using this https://elements.heroku.com/buildpacks/heroku/heroku-buildpack-multi-procfile

I was able to deploy them both as separate application

Reply:
As an example, it would be good to have this be an environment variable. As it would be in the actual implementation

Reply:
so something like `.env`, right?

Reply:
Here's how to do it for Heroku https://devcenter.heroku.com/articles/node-best-practices#be-environmentally-aware